Ghidra/Features/Base/src/main/java/ghidra/util/GhidraJarBuilder.java

```diff
@@ -338,12 +338,11 @@ public void buildSrcZip(File outputFile, TaskMonitor monitor)
 				wroteToZip |= writeZipRecursively(zip, srcDir.getAbsolutePath(), srcDir);
 			}
 		}
-		if (wroteToZip) {
-			System.out
-					.println("Can't create source zip!  Has source been downloaded and installed?");
-			// zip.close reports error if nothing has been written to it
-			zip.close();
+		if (!wroteToZip) {
+			System.out.println("Can't create source zip!  Has source been downloaded and installed?");
 		}
+		// zip.close reports error if nothing has been written to it
+		zip.close();
```

Comment:
The comment says that it should close if _nothing_ has been written to it, so if you're saying that's intentional then I'll switch it back to how it was

Reply:
`// zip.close reports error if nothing has been written to it`

My interpretation of that is we don't want to call `close()` if nothing has been written because it will produce an error.  Needless to say this code doesn't get run very often.

Reply:
I'll just try-catch it and print the stacktrace instead of letting the program die.

Reply:
No, nevermind, if you look up the callstack you can see an exception handler specifically for this.  I think the original writer was specifically trying to throw an exception here if we can't write to the zip.